src/nyc_trees/js/src/surveyPage.js

```diff
@@ -379,7 +379,7 @@ function submitSurveyWithTrees() {
         })
         .done(function(content) {
             window.alert('Successfully saved survey');
-            // TODO: Go to preview page
+            window.location.href = '/survey/detail/' + content.survey_id + '/';
```

Comment:
If you add an ``hasUnsavedData = false;`` before setting the href it will suppress the "are you sure you want to leave the page" message.